ci-operator/config/openshift/assisted-installer-agent/openshift-assisted-installer-agent-ocm-2.3.yaml

```diff
@@ -82,6 +82,7 @@ tests:
       ASSISTED_AGENT_IMAGE: pipeline:assisted-installer-agent
     env:
       ASSISTED_CONFIG: |
+        OPENSHIFT_VERSION=4.8
```

Comment:
remove this job entirely, it's not relevant to this branch

Reply:
different PR